tap_salesforce/salesforce/__init__.py

```diff
@@ -248,7 +248,7 @@ def __init__(
         self.rest_requests_attempted = 0
         self.jobs_completed = 0
         self.login_timer = None
-        self.data_url = "{}/services/data/v41.0/{}"
+        self.data_url = "{}/services/data/v48.0/{}"
```

Comment:
maybe add a global `API_VERSION` variable to make it clear that they are all using the same version.. Except if they are not of course?

Reply:
but these three api-version changes are in three different files, they can't use one global variables

Reply:
they can, you can import variable form some file in each 👍

Reply:
yeah, or be an argument to some overlying function thing that we define in the entry-function. But the pain is not great for just 3 files.